graphene_mongo/fields.py

```diff
@@ -72,15 +74,16 @@ def args(self, args):
 
     @property
     def field_args(self):
-        def is_filterable(kv):
-            return hasattr(kv[1], '_type') \
-                and callable(getattr(kv[1]._type, '_of_type', None))
-
-        return reduce(
-            lambda r, kv: r.update(
-                {kv[0]: kv[1]._type._of_type()}) or r if is_filterable(kv) else r,
-            self.fields.items(), {}
-        )
+        def is_filterable(v):
```

Comment:
You rocks!! 👍

Reply:
Thanks. I'm new to GraphQL and graphene. Are there more types other than Structures and Connections that we need to check for?